server/database/config/build.sql

```diff
@@ -0,0 +1,74 @@
+BEGIN;
+
+DROP TABLE IF EXISTS users,parent,teacher,student,teacher-student,subjects,subjects-teacher,
+activities,homework CASCADE;
+
+CREATE TABLE users(
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(255) UNIQUE, 
+    user_name VARCHAR(50),
+    password VARCHAR(255),
+);
+
+CREATE TABLE parent(
+    id SERIAL PRIMARY KEY,
+    parent_id INTEGER,
+    user_id INTEGER,
+    FOREIGN KEY (user_id) REFERENCES users(id)
+);
+
+CREATE TABLE teacher(
+    id SERIAL PRIMARY KEY,
+    first_name VARCHAR(255),
+    last_name VARCHAR(255),
+    user_id INTEGER,
+    FOREIGN KEY (user_id) REFERENCES users(id)
+);
+
+CREATE TABLE student(
+    id SERIAL PRIMARY KEY,
+    student_name VARCHAR(255),
+    class INTEGER,
+    parent_id INTEGER,
+    FOREIGN KEY (parent_id) REFERENCES parent(parent_id)
+);
+
+CREATE TABLE teacher-student(
+    id SERIAL PRIMARY KEY,
```

Comment:
indentation for all columns in table `teacher-student`
```suggestion
   id SERIAL PRIMARY KEY,
```